{"level":"info","ts":1660205659.2542799,"caller":"userService/main.go:29","msg":"Loaded config"}
{"level":"fatal","ts":1660205659.2676969,"caller":"db/mysql.go:42","msg":"error_database_connection_failure","errorCode":250014,"error":"dial tcp: lookup host.docker.internal: no such host","stacktrace":"userService/db.InitDatabase\n\t/Users/ashleigh.aathar/go/src/entry-task/services/userService/db/mysql.go:42\nmain.main\n\t/Users/ashleigh.aathar/go/src/entry-task/services/userService/main.go:32\nruntime.main\n\t/usr/local/Cellar/go/1.19/libexec/src/runtime/proc.go:250"}
{"level":"info","ts":1660205870.088547,"caller":"userService/main.go:29","msg":"Loaded config"}
{"level":"fatal","ts":1660205870.091251,"caller":"db/mysql.go:42","msg":"error_database_connection_failure","errorCode":250014,"error":"dial tcp: lookup host.docker.internal: no such host","stacktrace":"userService/db.InitDatabase\n\t/Users/ashleigh.aathar/go/src/entry-task/services/userService/db/mysql.go:42\nmain.main\n\t/Users/ashleigh.aathar/go/src/entry-task/services/userService/main.go:32\nruntime.main\n\t/usr/local/Cellar/go/1.19/libexec/src/runtime/proc.go:250"}
{"level":"info","ts":1660209899.216747,"caller":"userService/main.go:29","msg":"Loaded config"}
{"level":"fatal","ts":1660209899.219528,"caller":"db/mysql.go:42","msg":"error_database_connection_failure","errorCode":250014,"error":"dial tcp: lookup host.docker.internal: no such host","stacktrace":"userService/db.InitDatabase\n\t/Users/ashleigh.aathar/go/src/entry-task/services/userService/db/mysql.go:42\nmain.main\n\t/Users/ashleigh.aathar/go/src/entry-task/services/userService/main.go:32\nruntime.main\n\t/usr/local/Cellar/go/1.19/libexec/src/runtime/proc.go:250"}
